packages/stark-ui/src/modules/date-picker/components/date-picker.config.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Object containing the format of the dates displayed in the date picker
```

Comment:
I think we can rename this file to "date-format.constants.ts" since it is just about formats and not a config file.

And can you add the URL where we can find the accepted formats in moment that we can use?